plugins/outputs/timestream/sample.conf

```diff
@@ -115,3 +89,80 @@
   ## Specify the maximum number of parallel go routines to ingest/write data
   ## If not specified, defaulted to 1 go routines
   max_write_go_routines = 25
+
```

Comment:
Hi, thanks for updating the PR after reviews! I have another small one:

Could we move this section out of the toml and into README.md? The readme is the best place to put examples and longer sections like this. The toml comments should only be reminders of how to use specific variables.

We could leave a line like "see plugin readme for details" if you're worried people won't know to look in the readme.

Otherwise the PR looks good to me.

Reply:
Hi, 
Thanks for the suggestion, it makes sense. I just moved it under README.
readme_linter is failing for a different plugin, hope that doesn't block merging this PR.